### PR TITLE
Added skykit to the quickstart folder 

### PR DESCRIPTION
**Pull request by marty733**
# Summary

Adding Skykit to the quickstart folder. This quickstart does not have any components that will live within the New Relic software, it will only be on the marketing site. 

Note: review support Level, marked as verified per discussion with New Relic. 

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Are all documentation links publically accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [x] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/skykit/config.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# Documentation references
+documentation:
+  - name: Get Started With Skykit Dashboards 
+    url: https://www.skykit.com/products/dashboards/connections/new-relic/
+    description: Connect with Skykit's team to get started displaying your New Relic dashboards 
+
+# Content / Design
+icon: logo.png
+website: https://www.skykit.com/products/dashboards/connections/new-relic/
```
Comment (jcountsNR) [inline quickstarts/skykit/config.yml:44]:
```suggestion
website: https://www.skykit.com
```

This should just be the company site, you can leave the documentation link in the doc references.

**Review by jcountsNR — CHANGES_REQUESTED** [review]
I only saw one small issue with the URL, the rest looks good. I might defer to the review team on the language here and whether or not this would qualify as 'verified'.

**Comment by jcountsNR** [comment]
@marty733 , looks like you have not signed the contributor license agreement. Can you take a look above?

**Comment by marty733** [comment]
@jcountsNR Signed the CLA, let me know what else I can do to help!

**Review by jcountsNR — APPROVED** [review]

**Pull request merged**